package/MDAnalysis/analysis/contacts.py

```diff
@@ -397,10 +397,15 @@ def __init__(self, u, selection, refgroup, method="hard_cut", radius=4.5,
         self.u = u
         super(Contacts, self).__init__(self.u.trajectory, **basekwargs)
 
+        self.fraction_kwargs = kwargs if kwargs is not None else {}
+
         if method == 'hard_cut':
             self.fraction_contacts = hard_cut_q
         elif method == 'soft_cut':
             self.fraction_contacts = soft_cut_q
+        elif method == 'radius_cut':
+            self.fraction_contacts = radius_cut_q
```

Comment:
I think here it's cleaner to use [function currying](https://en.wikipedia.org/wiki/Currying).

If you do `self.fraction_contacts = functools.partial(radius_cut_q, radius=radius)` it should work the same

Reply:
Sure, done :-)